### ISSUE TITLE
Missing files in the 1.23.0 tarball after make dist

### ISSUE THREAD

**Issue by vbwagner** [pgbouncer/pgbouncer#1123]
Version 1.23.0 includes new files test/pgbouncer_hba.conf and test/pgident.conf which are neccessary to run regression tests.
But these files are not included in tarball created by make dist.
This causes problems when building rpm and deb packages because packaging pipeline is based on source tarballs, not on git repositories.

Fix is quote trivial
```
--- pgbouncer/test/Makefile.orig        2024-07-22 07:30:34.198277859 +0300
+++ pgbouncer/test/Makefile     2024-07-22 07:31:31.751847707 +0300
@@ -9,8 +9,8 @@
 DIST_SUBDIRS = ssl
 
 EXTRA_DIST = conntest.sh ctest6000.ini ctest7000.ini run-conntest.sh \
-            hba_test.eval hba_test.rules Makefile \
-            test.ini stress.py userlist.txt \
+            hba_test.eval hba_test.rules Makefile pgident.conf\
+            test.ini stress.py userlist.txt pgbouncer_hba.conf \
             __init__.py conftest.py utils.py \
             test_admin.py test_auth.py test_cancel.py test_copy.py test_limits.
py \
             test_misc.py test_no_database.py test_no_user.py test_operations.py
 \
```

**Comment by JelteF** [comment]
Could you create a PR for this? Then we'll merge it and release it as part of the upcoming 1.23.1

**Comment by JelteF** [comment]
Fixed by #1124
